.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,116 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
remove or ignore this file?

Reply:
These are IntelliJ files, the whole .idea/ directory needs to be ignored

Reply:
weird, I through we ignore IntelliJ files already